<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTerminologyEditorWidget</class>
 <widget class="QWidget" name="qSlicerTerminologyEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qSlicerTerminologyEditorWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Value</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Scheme</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Region:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="typeModifierCodeMeaningLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="typeModifierCSDLineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="categoryCodeMeaningLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>    Modifier:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="categoryCSDLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="typeCSDLineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="categoryCodeValueLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="typeCodeMeaningLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="typeCodeValueLineEdit"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="typeModifierCodeValueLineEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Meaning</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>    Modifier:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="regionModifierCodeMeaningLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="regionModifierCodeValueLineEdit"/>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="regionModifierCSDLineEdit"/>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="regionCodeValueLineEdit"/>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="regionCSDLineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="regionCodeMeaningLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="autoGeneratedInfoFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QCheckBox" name="useAutoGeneratedCheckBox">
        <property name="text">
         <string>Use automatically generated label and color from selected terminology:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>12</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="autoGeneratedColorLabel"/>
        </item>
        <item>
         <widget class="QLabel" name="autoGeneratedNameLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="selectFromTerminologyButton">
     <property name="text">
      <string>Select from terminology...</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>categoryCodeMeaningLineEdit</tabstop>
  <tabstop>categoryCodeValueLineEdit</tabstop>
  <tabstop>categoryCSDLineEdit</tabstop>
  <tabstop>typeCodeMeaningLineEdit</tabstop>
  <tabstop>typeCodeValueLineEdit</tabstop>
  <tabstop>typeCSDLineEdit</tabstop>
  <tabstop>typeModifierCodeMeaningLineEdit</tabstop>
  <tabstop>typeModifierCodeValueLineEdit</tabstop>
  <tabstop>typeModifierCSDLineEdit</tabstop>
  <tabstop>regionCodeMeaningLineEdit</tabstop>
  <tabstop>regionCodeValueLineEdit</tabstop>
  <tabstop>regionCSDLineEdit</tabstop>
  <tabstop>regionModifierCodeMeaningLineEdit</tabstop>
  <tabstop>regionModifierCodeValueLineEdit</tabstop>
  <tabstop>regionModifierCSDLineEdit</tabstop>
  <tabstop>selectFromTerminologyButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../Terminologies/Widgets/Resources/qSlicerTerminologiesModuleWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
